Userland/Libraries/LibCore/System.h

```diff
@@ -51,7 +51,7 @@
 namespace Core::System {
 
 #ifdef AK_OS_SERENITY
-ErrorOr<void> beep(Optional<size_t> tone);
+ErrorOr<void> beep(Optional<size_t> tone, Optional<size_t> milliseconds_duration);
```

Comment:
```suggestion
ErrorOr<void> beep(Optional<size_t> tone, Optional<size_t> milliseconds_duration = {});
```

That said, `Optional<>` doesn't make much sense here, does it?

```suggestion
ErrorOr<void> beep(size_t tone = 440, size_t milliseconds_duration = 200);
```